homeassistant/components/lg_netcast/media_player.py

```diff
@@ -185,7 +185,11 @@ def supported_features(self):
     @property
     def media_image_url(self):
         """URL for obtaining a screen capture."""
-        return self._client.url + "data?target=screen_image"
+        return (
+            self._client.url
+            + "data?target=screen_image&_="
+            + str(int(datetime.now().timestamp()))
```

Comment:
Please use f-strings as mentioned in the [style guidelines](https://developers.home-assistant.io/docs/en/development_guidelines.html).

Reply:
f-strings problem fixed.
What kind of documentation required and where to add the link?